include/zephyr/modbus/modbus.h

```diff
@@ -45,16 +45,16 @@ extern "C" {
  *  @{
  */
 /* Modbus exception codes */
-#define MODBUS_EXC_NONE				0
-#define MODBUS_EXC_ILLEGAL_FC			1
-#define MODBUS_EXC_ILLEGAL_DATA_ADDR		2
-#define MODBUS_EXC_ILLEGAL_DATA_VAL		3
-#define MODBUS_EXC_SERVER_DEVICE_FAILURE	4
-#define MODBUS_EXC_ACK				5
-#define MODBUS_EXC_SERVER_DEVICE_BUSY		6
-#define MODBUS_EXC_MEM_PARITY_ERROR		8
-#define MODBUS_EXC_GW_PATH_UNAVAILABLE		10
-#define MODBUS_EXC_GW_TARGET_FAILED_TO_RESP	11
+#define MODBUS_EXC_NONE				0    /**< No exception */
+#define MODBUS_EXC_ILLEGAL_FC			1    /**< Illegal function code */
+#define MODBUS_EXC_ILLEGAL_DATA_ADDR		2    /**< Illegal data address */
+#define MODBUS_EXC_ILLEGAL_DATA_VAL		3    /**< Illegal data value */
+#define MODBUS_EXC_SERVER_DEVICE_FAILURE	4    /**< Server device failure */
+#define MODBUS_EXC_ACK				5    /**< Acknowledge */
+#define MODBUS_EXC_SERVER_DEVICE_BUSY		6    /**< Server device busy */
+#define MODBUS_EXC_MEM_PARITY_ERROR		8    /**< Memory parity error */
+#define MODBUS_EXC_GW_PATH_UNAVAILABLE		10   /**< Gateway path unavailable */
+#define MODBUS_EXC_GW_TARGET_FAILED_TO_RESP	11   /**< Gateway target device failed to respond */
```

Comment:
The length of this line is at the limit, this style does not scale, please find another way (I have already asked you) to document this.

Reply:
updated, thx!

Reply:
@kartben Thank you. There are now two typos, `#define MODBUS_EXC_GW_TARGET_FAILED_TO_RESP	1`..., please recheck.

Reply:
That was really bad, sorry!